emulator_rubocop.gemspec

```
@@ -1,39 +1,27 @@
 # frozen_string_literal: true
 
-require_relative "lib/emulator_rubocop/version"
+require_relative "lib/modules/version"
```

Please use single quotes here and everywhere in the code
```suggestion
require_relative 'lib/modules/version'
```

---

Fixed